frontend/src/lib/api.ts

```diff
@@ -778,7 +779,7 @@ const api = {
             limit?: number
             offset?: number
             teamId?: TeamType['id']
-            type?: 'event' | 'person' | 'group'
+            type?: string
```

Comment:
What new types are possible outside of `'event' | 'person' | 'group'`?

Reply:
Sure! Here are 21 more changed files. 😅